examples/.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+    'extends': [
+        'eslint-config-airbnb-base',
+        'eslint-config-airbnb-base/rules/strict',
+    ],
+    parserOptions: {
+        ecmaVersion: 5,
+        sourceType: 'script',
+        ecmaFeatures: {
+            impliedStrict: true
+        }
+    },
+    env: {
+        browser: true,
+        es6: false,
+        amd: true,
+        commonjs: true
+    },
+    rules: {
+        'no-plusplus': 'off',
+        // this option sets a specific tab width for your code
+        // http://eslint.org/docs/rules/indent
+        indent: ['error', 4, {
+            SwitchCase: 1,
+            VariableDeclarator: 1,
+            outerIIFEBody: 1,
+            // MemberExpression: null,
+            // CallExpression: {
+            // parameters: null,
+            // },
+            FunctionDeclaration: {
+                parameters: 1,
+                body: 1
+            },
+            FunctionExpression: {
+                parameters: 1,
+                body: 1
+            }
+        }],
+        'one-var': ['error', 'never'],
+        'valid-jsdoc': ['error', {
+            requireReturn: false,
+            requireParamDescription: false,
+            requireReturnDescription: false,
+        }],
+
+        // deactivated rules for es5
+        'no-var': 'off',
+        'prefer-arrow-callback': 'off',
+        'object-shorthand': 'off',
+        'no-param-reassign': ['error', { 'props': false }],
+        'no-mixed-operators': ['error', { allowSamePrecedence: true }],
+
+        // TODO reactivate all the following rules
+        'no-underscore-dangle': 'off',
+
```

Comment:
There's no `"root": true` in this file, so it's merged with the _project global_ `.eslintrc.js`, which means those _overrides_ are useless, and the removed lines here do not _reactivate_ the corresponding rules.

We should probably go for simplicity here and simply override the `parserOptions` and `env`.